boards/xtensa/esp32/common/include/esp32_bmp180.h

```diff
@@ -0,0 +1,84 @@
+/****************************************************************************
+ * boards/arm/esp32/common/src/esp32_bmp180.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ESP32_BMP180_H
+#define __ESP32_BMP180_H
```

Comment:
```suggestion
#ifndef __ARCH_XTENSA_SRC_ESP32_BMP180_H
#define __ARCH_XTENSA_SRC_ESP32_BMP180_H
```
This header guard is not conforming to NuttX coding standards.
Please, take a look at the [Forming Guard Names] section (https://cwiki.apache.org/confluence/display/NUTTX/Coding+Standard#main-content:~:text=Forming%20Guard%20Names.%20Then%20pre%2Dprocessor%20macro,corresponds%20to%20the%20header%20file%20include%2Fnuttx%2Farch.h)

Reply:
```
#ifndef __BOARDS_XTENSA_ESP32_COMMON_INCLUDE_BOARD_BMP180_H
#define __BOARDS_XTENSA_ESP32_COMMON_INCLUDE_BOARD_BMP180_H
```
To add on Gustavo's suggestion.